onnxruntime/core/providers/coreml/builders/helper.h

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 #include <stdint.h>
+#include <functional>
 
```

Comment:
Why do we need to add the include of 'functional' here if there are no other changes in this file? 
nit: stdint.h -> cstdint